python/DEV_SETUP.md

```diff
@@ -284,5 +215,38 @@ To run the same checks that run during the GitHub Action build, you can use
 this command, from the [python](../python) folder:
 
 ```bash
-    poetry run pre-commit run -c ../.pre-commit-config.yaml -a
+    poetry run pre-commit run -a
 ```
+
+or use the following task (using `Ctrl+Shift+P`):
+- `Python - Run Checks` to run the checks on the whole project.
+- `Python - Run Checks - Staged` to run the checks on the currently staged files only.
+
+Ideally you should run these checks before committing any changes, use `poetry run pre-commit install` to set that up.
+
+## Code Coverage
+
+We try to maintain a high code coverage for the project. To run the code coverage on the unit tests, you can use the following command:
+
+```bash
+    cd python
+    poetry run pytest --cov=semantic_kernel --cov-report=term-missing:skip-covered tests/unit/
+```
+or use the following task (using `Ctrl+Shift+P`):
+- `Python: Tests - Code Coverage` to run the code coverage on the whole project.
+
+This will show you which files are not covered by the tests, including the specific lines not covered.
+
+## Catching up with the latest changes
+There are many people committing to Semantic Kernel, so it is important to keep your local repository up to date. To do this, you can run the following commands:
+
+```bash
```

Comment:
Since all changes are merge squash, this is also an acceptable operation:

```
git fetch upstream
git merge upstream/main
git push origin main
```